webui/react/design/index.tsx

```diff
@@ -21,21 +22,26 @@ const fakeSettingsContext = {
   state: observable(Map<string, Settings>()),
 };
 
+const router = createBrowserRouter([
+  {
+    path: "*",
+    element: <HelmetProvider>
+              <UIProvider>
+                <UserSettings.Provider value={fakeSettingsContext}>
+                  <ThemeProvider>
+                    <ConfirmationProvider>
+                      <div className={css.base}>
+                        <DesignKit />
+                      </div>
+                    </ConfirmationProvider>
+                  </ThemeProvider>
+                </UserSettings.Provider>
+              </UIProvider>
+            </HelmetProvider>,
+  },
+]);
+
 // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 createRoot(document.getElementById('root')!).render(
-  <BrowserRouter>
-    <HelmetProvider>
-      <UIProvider>
-        <UserSettings.Provider value={fakeSettingsContext}>
-          <ThemeProvider>
-            <ConfirmationProvider>
-              <div className={css.base}>
-                <DesignKit />
-              </div>
-            </ConfirmationProvider>
-          </ThemeProvider>
-        </UserSettings.Provider>
-      </UIProvider>
-    </HelmetProvider>
-  </BrowserRouter>,
+  <RouterProvider router={router} />
```

Comment:
Since we use `useBlocker` in notes, we need to update the router component with  `createBrowserRouter`